### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by sanketsudake**

### CODE REVIEW

**Comment by sanketsudake** [comment]
@pravarag Please add content in README about how to use this project.

File: Charts/jaeger-helm/jaeger-helm/templates/#jaeger_dep_svc.yaml# [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
```
Comment (sanketsudake) [inline Charts/jaeger-helm/jaeger-helm/templates/#jaeger_dep_svc.yaml#]: Remove unwanted file

File: Charts/jaeger-helm/jaeger-helm/new_file.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:                     app1-service
```
Comment (sanketsudake) [inline Charts/jaeger-helm/jaeger-helm/new_file.yaml]: Whats purpose of this file ?

File: Charts/jaeger-helm/jaeger-helm/templates/.#jaeger_dep_svc.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+pravar@localhost.localdomain.26423:1545376380
```
Comment (sanketsudake) [inline Charts/jaeger-helm/jaeger-helm/templates/.#jaeger_dep_svc.yaml]: Remove

File: Charts/jaeger-helm/jaeger-helm/templates/old_yamls/jaeger_operator_deploy.old [hunk 4]
```diff
@@ -0,0 +1,27 @@
+apiVersion: "io.jaegertracing/v1alpha1"
```
Comment (sanketsudake) [inline Charts/jaeger-helm/jaeger-helm/templates/old_yamls/jaeger_operator_deploy.old]: Remove

File: Charts/jaeger-helm/jaeger-helm/templates/old_yamls/my_jaeger_svc.old [hunk 5]
```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```
Comment (sanketsudake) [inline Charts/jaeger-helm/jaeger-helm/templates/old_yamls/my_jaeger_svc.old]: Remove

File: Charts/jaeger-helm/templates/old_yamls/jaeger_operator_deploy.old [hunk 6]
```diff
@@ -0,0 +1,27 @@
+apiVersion: "io.jaegertracing/v1alpha1"
```
Comment (sanketsudake) [inline Charts/jaeger-helm/templates/old_yamls/jaeger_operator_deploy.old]: remove

File: Charts/jaeger-helm/templates/old_yamls/my_jaeger_svc.old [hunk 7]
```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```
Comment (sanketsudake) [inline Charts/jaeger-helm/templates/old_yamls/my_jaeger_svc.old]: remove

File: Charts/jaeger-helm/jaeger-helm/new_file.yaml (hunk 2 above)
Comment (pravarag) [reply Charts/jaeger-helm/jaeger-helm/new_file.yaml]: file removed

File: Charts/jaeger-helm/jaeger-helm/templates/old_yamls/my_jaeger_svc.old (hunk 5 above)
Comment (pravarag) [reply Charts/jaeger-helm/jaeger-helm/templates/old_yamls/my_jaeger_svc.old]: file removed

**Pull request merged**